### PR TITLE
Update library dependencies.

### PR DESCRIPTION
**Pull request by schleichardt**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -28,8 +28,6 @@ autoScalaLibrary in ThisBuild := false // no dependency to Scala standard librar
 
 crossPaths in ThisBuild := false
 
-Release.publishSettings
```
Comment (lauraluiz) [inline build.sbt]: What did it do?

**Comment by lauraluiz** [comment]
Good.

**Comment by lauraluiz** [comment]
But check the build error.

File: build.sbt (hunk 1 above)
Comment (schleichardt) [inline build.sbt]: It sets the publish settings to sonatype, but was duplicated.

Comment (schleichardt) [inline build.sbt]: see line 49/47

**Pull request merged**